source/elements/oneTBB/source/flow_graph/limiter_node_cls.rst

```diff
@@ -54,8 +54,11 @@ to the decrementer determine the value by which the internal counter of broadcas
 will be decreased, while negative values determine the value by which the internal
 counter of broadcasts will be increased.
 
-The ``continue_msg`` sent to the decrementer
-decreases the internal counter of broadcasts by one.
+If ``continue_msg`` is used as an argument for ``DecrementType`` template
+parameter then the ``decrementer``'s port of the ``limiter_node`` also acquires the
+behavior of the continue_node, requiring the number of messages sent to it
```

Comment:
```suggestion
behavior of the ``continue_node``. This behavior requires the number of messages sent to it
```

Reply:
Not sure what here **requires the number...** . If it's not the behavior but ``continue_node``, try just **It requires the...**